include/calibrationLib.py

```diff
@@ -1,10 +1,13 @@
 # calibrationLib.py
-#
+#GVF Source: https://medium.com/analytics-vidhya/jenks-natural-breaks-best-range-finder-algorithm-8d1907192051
```

Comment:
Reusing code is _very_ common in the programming community, the same block appeared back in 2015 on CrossValidated: https://stats.stackexchange.com/a/144075